MixItUp.WPF/Controls/Command/CommandButtonsControl.xaml.cs

```diff
@@ -134,7 +134,36 @@ private void RefreshUI()
             CommandBase command = this.GetCommandFromCommandButtons<CommandBase>(this);
             if (command != null)
             {
-                await command.PerformAndWait(ChannelSession.GetCurrentUser(), new List<string>() { "@" + ChannelSession.GetCurrentUser().UserName });
+                Dictionary<string, string> extraSpecialIdentifiers = null;
+                if (command is EventCommand)
+                {
+                    EventCommand eventCommand = command as EventCommand;
+                    switch (eventCommand.OtherEventType)
+                    {
+                        case OtherEventTypeEnum.GameWispSubscribed:
+                        case OtherEventTypeEnum.GameWispResubscribed:
+                            extraSpecialIdentifiers = new Dictionary<string, string>
+                            {
+                                { "subscribemonths", "999" },
+                                { "subscribetier", "Test Tier" },
+                                { "subscribeamount", "$12.34" },
+                            };
+                            break;
+                        case OtherEventTypeEnum.StreamlabsDonation:
+                        case OtherEventTypeEnum.GawkBoxDonation:
+                        case OtherEventTypeEnum.TiltifyDonation:
+                            extraSpecialIdentifiers = new Dictionary<string, string>
+                            {
+                                { "donationsource", "Test Source" },
+                                { "donationamount", "$12.34" },
+                                { "donationmessage", "Test donation message." },
+                                { "donationimage", ChannelSession.GetCurrentUser().AvatarLink },
+                            };
+                            break;
+                    }
```

Comment:
Looks good! Only thing I might add are default values for some of these ones as well:

- Host event: $hostviewercount
- Resub event: $usersubmonths

For the $usersubmonths, i'm not sure if manually setting a value here will overwrite whatever might be returned from the normal $usersubmonths look up, but it'd be interesting to try.